.github/workflows/clear-artifacts.yml

```diff
@@ -11,7 +11,7 @@ on:
         type: string
```

Comment:
The release branch code is already merged to develop-java21 branch by merging the develop branch code. Since we need to correct the version to 1.2.1-SNAPSHOT from 1.2.1-java21-SNAPSOT, the fork was used instead of directly merging.